packages/engine/src/avatar/animation/AvatarAnimationGraph.ts

```diff
@@ -29,7 +29,7 @@ const getDistanceAction = (animationName: string, mixer: AnimationMixer): Distan
 
 /** Class to hold the animation graph for player entity. Every avatar entity will have their saperate graph. */
 export class AvatarAnimationGraph extends AnimationGraph {
-  initialize(mixer: AnimationMixer, velocity: Vector3, avatar: AvatarComponentType) {
+  initialize(mixer: AnimationMixer, velocity: Vector3, jumpValue: {} | null = null) {
```

Comment:
If the jumpValue is a boolean, then it will get passed as a copy, not a reference. Another way I can think of to get this working is to get the component inside the initialized function instead of passing it. Or update the graph's jump property in a system.